src/components/shared/Header/Header.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { images, menuItems } from "../../../data/header";
+import "./Header.css";
+
+class Header extends Component {
+  state = {
+    isDisplayed: false,
+    menuItems,
+  };
+
+  /**
+   * Checks the active value to add class "active"
+   * @param {String} active
+   * @param {String} item
+   * @returns {Boolean} If active value equal item name, return "active", else null
+   */
+  isActive = (active, item) => {
+    return active === item ? "active" : "";
+  };
+
+  /**
+   * Toggle display value
+   */
+  onMenu = () => {
+    this.setState({ isDisplayed: !this.state.isDisplayed });
+  };
+
+  /**
+   * Checks display value to add class "display-toggle".
+   * @returns {Boolean} If the display is true, return "display-toggle", else null.
+   */
+  toggleDisplay = () => {
+    return this.state.isDisplayed === true ? "display-toggle" : "";
+  };
+
+  render() {
+    const { active, onActive } = this.props;
+    return (
```

Comment:
Missing line space before return
```suggestion

    return (
```